components/ResourceCard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import Image from 'next/image'
+
+const BadgeContainer = styled.div`
+  display: flex; 
+  position: absolute; 
+  margin-top: -30px; 
+  margin-left: -30px;
```

Comment:
If it is position absolute does this work?
```suggestion
  top: -30px; 
  left: -30px;
```

Reply:
i cannot get it to work :( i also believe that becomes relative to the entire container which is a bit trickier and less intuitive

Reply:
oh yeah it becomes absolutely positioned relative to its closest "position relative" parent. So set the parent to position: relative I believe

Reply:
word it works!